Bottom: 63d9ee86b7812695228cc4f4018bf998f3eaaa39
Top:    e2c0014c734beb600d34d764ea62c999ea687c38
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-02-04 16:52:18 -0800

librdmacm/rstream: Reduce default transfer count

1 million ping-pong transfers takes over 3 seconds to
complete, and I'm impatient.  Reduce the default number of
transfers for small messsages to speed up running
performance tests, especially when running over slower
connections, like TCP sockets or over a WAN.

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/examples/common.c b/examples/common.c
index 939f50c..294dba1 100644
--- a/examples/common.c
+++ b/examples/common.c
@@ -123,10 +123,8 @@ int size_to_count(int size)
 		return 1000;
 	else if (size >= 10000)
 		return 10000;
-	else if (size >= 1000)
-		return 100000;
 	else
-		return 1000000;
+		return 100000;
 }
 
 void format_buf(void *buf, int size)
